src/game_api/script/usertypes/vanilla_render_lua.cpp

```diff
@@ -285,6 +294,7 @@ void register_usertypes(sol::state& lua)
 
     lua.new_usertype<TextRenderingInfo>(
         "TextRenderingInfo",
+        // sol::constructors<TextRenderingInfo(const std::string, float, float, float, float, uint32_t, uint32_t)>{},
```

Comment:
not sure what I'm doing wrong, but this just insta crash when you try to constructs new one